### PR TITLE
An initial r-tidyverse image for the public beta

### PR DESCRIPTION
**Pull request by davclark**

### CODE REVIEW

**Comment by davclark** [comment]
Don't merge until next app update

File: base/r-tidyverse/r-tidyverse_r0.yaml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
...cDo1t22ZQiTYVttguxHZNhdMIbkB7jw2j9dcfxcuevxXgCN7pzi4YCKeF4GqAH2bpmWfERNBkpxNG2xo4UhZHGoWiTiDrTIguxA10G7T2I1JhvrIuFX1ruMqzu2124+9j2yDbE8xAFaHGRbyCjc8/ctAldf9fhqQdQA2ZgCz7K9rq+xWOo9gGhOBYirxFrWynUD1JSCr1Mo5zzS0Qi+v+xxZ2FeIlUmbqSi6jbdD4pmwJE1E3UWaSA6YT8PQN6ZF3JZpZBtke4gBYAzhpvuX8eYXfAmPvfLLwOELEkd3il6msVkrsen/xAY72ZkWP+xkKwEs4AKtxngr1NbwbJcIY+kT4nUW6SXjJjl/koWCFwnlSThHdUOnaQEAXdafmW50hfeRJdsgmyg1wAI3HdiJ737q/fiZV/53YGV5cghtz56nehHI3CTYdDEbzsiM6HwJ5oRzDrBWLGuXzJ7boENn2sYYrJS+RxYiUZ/Un2npd7jfefo9gU4f2I9PzdC3gTZJthe+EsQy2M/etxPfeuWD+P0ffh8wGAEnl6e0xcqVpZA3gRUOLJodVrpeyfS2KsDKa+k9wCVj/wD/XIZpB7oUxZAdWdOVO/y4RECp+S7XRgA2dn5sZ8AUsr0gliLbTLZDLMDefGAXXvLEQ/iDN/wVkBHw4D4gr5A+mmsdtcsRsYR+2r3msmWzCQGs4geLjuzPCoOVjCgF0ASbDXpPBKTXZ2PQA2D9n32ycSUGmD0BNp7hBokWX1PYFk22mWxELMB++t7deNE1h/EHb/xLIC+Bo+dNALaHeCYCDrABMAt2Vs540P6Odr4+ABvIfCYmAvYconUOZd3HY77qAbA2/b7liVXC2otl+pK9Rrqlurutjx12m32ebdlmsgFpTAT378SLrz6Md77ufQBK4ND5QD7uRxwaYXZWFWz5b415hqb9FGFz05yopcimmgjkTbmTKySzqkNAuReDTa0DZ+DyXofvszqgbBhgN4mCbjPakGyDrCKNF8F9u/DSaw7iD17/F4CpgEP7gGI0HRlo2l5CZHW7LBBkpTMFm5j+NADbtVUWiHsH8O9ZpQJz1pPBpgKslVSAZWXqvchlB+bYub8yTvCCi9lUXNwmyp5sg6wQ60Xw2ft34YVXHsYfvPa9QDYGHtwPFOMNpMy/+xWZTpLVDaUB+ItcsV6yFWywVoz4q0XoAFgnToqr
```
Comment (davclark) [inline base/r-tidyverse/r-tidyverse_r0.yaml]: Need to fix this Icon!

File: base/r-tidyverse/r-tidyverse_r0.yaml [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+os_release: "18.04"
+license: MIT
+url: null
+image:
+  server: hub.docker.com
+  namespace: gigantum
+  repository: r-tidyverse
+  tag: 376dd729-2018-06-05
+languages:
+  - python3
```
Comment (davclark) [inline base/r-tidyverse/r-tidyverse_r0.yaml]: If we aren't able to fix the UI in time for 0.8.2 (or whatever), we can comment this line

File: base/r-tidyverse/r-tidyverse_r0.yaml [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  - rstats
```
Comment (dkleissa) [inline base/r-tidyverse/r-tidyverse_r0.yaml]: `rstats` vs. `R`. How do we want it to render for users?

**Pull request merged**